mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -181,20 +159,15 @@ def run_create_tutorial(input_file) -> None:
             1: build_area_tutorial.create_tutorial,
             3: change_detection_tutorial.create_tutorial,
         }
-
         project_types_tutorial[project_type](tutorial)
-    except Exception as e:
-        slack.send_error(e)
-        sentry.capture_exception_sentry(e)
-        logger.exception(e)
+    except Exception:
+        logger.exception()
+        sentry.capture_exception()
 
 
 @click.command("archive")
 @click.option(
-    "--project-id",
-    "-i",
-    help=("Archive project with giving project id"),
-    type=str,
+    "--project-id", "-i", help=("Archive project with giving project id"), type=str,
 )
 @click.option(
     "--project-ids",
```

Comment:
I'm a bit confused that we have both options (project-id and project-ids). But I think this is fine, just need to tell users how to use it properly in the docs.

Reply:
I have let this option in the code because I find the list option kind of unintuitive. If one wants to archive only one project one has to write: `mapswipe_workers archive --project-ids ["my_project_id"]` instead of simply ``mapsiwpe_workers archive --project_id my_project_id`.

Maby it is possible to have on option with two types of input. Either a list or simply a string.

Should we open an issue for that? This would apply also to the other cli arguments such as transfer results etc.

Reply:
let's keep it as you did. I think we don't need another issue for this.